pkg/manager/manager.go

```diff
@@ -117,3 +118,41 @@ type manager struct {
 func (m *manager) GetContext() *context.ControllerManagerContext {
 	return m.ctx
 }
+
+func UpdateCredentials(opts *Options) {
+	opts.updateCredentials()
+}
+
+// initializeWatch is a filesystem watcher for the capv credentials file
+// pass the new credentials to the capv manager context whenever there is an update
+// to the file.
```

Comment:
```suggestion
// InitializeWatch adds a filesystem watcher for the capv credentials file
// In case of any update to the credentials file, the new credentials are passed to the capv manager context.
```